src/main/java/nl/tudelft/contextproject/model/entities/VRPlayer.java

```diff
@@ -1,261 +1,258 @@
-			bomb.move((int) vec.x, (int) vec.y + 1, (int) vec.z);
-			if (Main.getInstance().getCurrentGame() != null) {
-				Main.getInstance().getCurrentGame().addEntity(bomb);
-			}
-		}
-	}
-	
-	/**
-	 * Player picks up a nearby item.
-	 * Also opens nearby doors if possible
-	 */
-	public void pickUp() {
-		Set<Entity> set = Main.getInstance().getCurrentGame().getEntities();
-		for (Entity ent : set) {
-			 if (ent.collidesWithPlayer(2f)) {
-				if (ent instanceof Bomb) {
-					inventory.add(new Bomb());
-					ent.setState(EntityState.DEAD);
-					return;
-				}
-				if (ent instanceof Key) {
-					Key key = (Key) ent;
-					inventory.add(new Key(key.getColor()));
-					ent.setState(EntityState.DEAD);
-					return;
-				}
-				if (ent instanceof Door) {
-					Door door = (Door) ent;
-					if (inventory.containsColorKey(door.getColor())) {
-						inventory.remove(inventory.getKey(door.getColor()));
-						ent.setState(EntityState.DEAD);
-						return;
-					}
-				}
-			}
-		}
-	}
-
-	@Override
-	public void move(float x, float y, float z) {
-		spatial.move(x, y, z);
-		getPhysicsObject().setPhysicsLocation(playerControl.getPhysicsLocation().add(x, y, z));
-	}
-
-	/**
-	 * Method used for testing.
-	 * Returns the player's inventory.
-	 * @return The player's inventory
-	 */
-	public Inventory getInventory() {
-		return inventory;
-	}
-
-	/**
-	 * Method used for testing.
-	 * Sets a player's inventory
-	 * @param inv
-	 * 		Inventory to be set
-	 */
-	public void setInventory(Inventory inv) {
-		inventory = inv;
-	}
-}
+package nl.tudelft.contextproject.model.entities;
+
+import java.awt.Graphics2D;
+
+import java.util.Set;
+
+import com.jme3.bullet.collision.shapes.CapsuleCollisionShape;
+import com.jme3.bullet.control.CharacterControl;
+import com.jme3.input.controls.ActionListener;
+import com.jme3.material.Material;
+import com.jme3.math.ColorRGBA;
+import com.jme3.math.Vector3f;
+import com.jme3.scene.Geometry;
+import com.jme3.scene.Spatial;
+import com.jme3.scene.shape.Sphere;
+
+import nl.tudelft.contextproject.Main;
+import nl.tudelft.contextproject.model.Inventory;
+import nl.tudelft.contextproject.model.PhysicsObject;
+
+/**
+ * Class representing the player wearing the VR headset.
+ */
+public class VRPlayer extends Entity implements ActionListener, PhysicsObject {
+	/**
+	 * Physics interaction constants.
+	 */
+	public static final float JUMP_SPEED = 13f;
+	public static final float FALL_SPEED = 15f;
+	public static final float PLAYER_GRAVITY = 13f;
+
+	/**
+	 * Physical collision model.
+	 */
+	public static final float PLAYER_STEP_HEIGHT = 0.1f;
+	public static final float PLAYER_RADIUS = .5f;
+	public static final float PLAYER_HEIGHT = 3f;
+	//SHOULD NOT BE CHANGED
+	public static final int PLAYER_AXIS = 1;
+
+	/**
+	 * Movement control constants.
+	 */
+	public static final float SIDE_WAY_SPEED_MULTIPLIER = .08f;
+	public static final float STRAIGHT_SPEED_MULTIPLIER = .1f;
+
```

Comment:
I think that some of the comments above the constants provided additional, useful information, and should be readded.
E.g. PLAYER_GRAVITY "how fast the player accelerates while falling" is useful.